lib/smart_proxy_host_reports/ansible_processor.rb

```diff
@@ -61,4 +71,49 @@ def spool_report
     end
     SpooledHttpClient.instance.spool(report_id, payload)
   end
+
+  private
+
+  def log_level(log)
+    if log["failed"]
+      "err"
+    elsif log["changed"]
+      "notice"
+    else
+      "info"
+    end
+  end
+
+  def log_source(log)
+    # Temporary until  is resolved
+    log["module"]
+  end
+
+  def log_message(log)
+    return log["msg"] if log["failed"]
+
+    case log["module"]
```

Comment:
This whole method I've extraced from https://github.com/theforeman/foreman_ansible/blob/1c121a9345f5d19ebde14a7c61685f31e6801696/app/helpers/foreman_ansible/ansible_reports_helper.rb#L28. The main difference is that the strings will not be translated (`_()` is not availabe here, but maybe I missed something) and I try to add an error message if something failed.